### ISSUE TITLE
retinanet_x101 training NaN

### ISSUE THREAD

**Issue by liangzz1991** [open-mmlab/mmdetection#1278]
i used faster_rcnn_r101 is ok,,,, but when i try the retinanet_x101 is always become NaN for loss suddenly, because loss is falling .i feel weird~~I have tried a lot of lr but the same result in the 1st epoch .
<img width="856" alt="微信图片_20190827164441" src="https://user-images.githubusercontent.com/12033811/63756991-e799d900-c8eb-11e9-9cdf-fe6e367fb53d.png">

**Comment by hellock** [comment]
Has you followed the linear scaling rule in the documentation?

**Comment by liangzz1991** [comment]
sure，i set lr = 0.015, i have 6 gpus *2img/gpu,,,In addition, I have tried a lot of lr from1e-5 to 2e-2, but always become nan in 1st epoch，I confirm that the model of faster_rcnn101.py is ok..so i  feel weird~~no matter how lr is but the same result in the 1st epoch.

**Comment by liangzz1991** [comment]
@hellock what advice do you have for this question?

**Comment by wikiwen** [comment]
@liangzz1991 
I met a similar problem with you recently. 
I change batch_size from 4 to 8. Single gpu, lr 0.01
But no matter how I change lr, it always be nan after some steps.
In the end, I chang batch_size back to 4, lr to 0.005.
Problem disapperaed~~
It seems that big batch_size will cause this problem.

**Comment by liangzz1991** [comment]
@wikiwen Unfortunately，it does not work for my task. The object of my task has muti-scale and ratio, and very different. The model of faster rcnn is ok, and the other model has low MAP, but the mmdet-retinanet hard to converge, and I have tried a lot of hyperparameters.

**Comment by liangzz1991** [comment]
Accidentally found the reason of my problem ！~~there are some noisy in my dataset(x/y>w/h in xml or x/y<0 in xml). At that time, no problem when train used faster rcnn config, so I didn't think about the reason for the data when train retinanet.
